nmrfx-processor/src/main/java/org/nmrfx/processor/processing/MultiVecCounter.java

```diff
@@ -181,31 +178,31 @@ void init(int[] tdSizes, int[] outSizes, boolean[] complex, String[] modes) {
             for (int i = 0; i < outPhases.length; i++) {
                 System.out.print("ouPh[" + i + "]=" + outPhases[i] + " ");
             }
-            System.out.println("");
+            System.out.println();
```

Comment:
This entire section should be converted to log.debug, StringBuilder can be used to generate the logmessages for replacing the print in loops that are print instead of println